src/applications/claims-status/tests/components/ClaimsDecision.unit.spec.jsx

```diff
@@ -7,16 +7,18 @@ import ClaimsDecision from '../../components/ClaimsDecision';
 describe('<ClaimsDecision>', () => {
   it('should render message without date sentence', () => {
     const date = null;
+    const eBenefitsUrl = path => path;
     const tree = SkinDeep.shallowRender(
-      <ClaimsDecision completedDate={date} />,
+      <ClaimsDecision completedDate={date} eBenefitsUrl={eBenefitsUrl} />,
     );
 
     expect(tree.text()).not.to.contain('We finished reviewing your claim');
   });
   it('should render message with date sentence', () => {
     const date = '2010-03-01';
+    const eBenefitsUrl = path => path;
     const tree = SkinDeep.shallowRender(
-      <ClaimsDecision completedDate={date} />,
+      <ClaimsDecision completedDate={date} eBenefitsUrl={eBenefitsUrl} />,
```

Comment:
Are these references to `eBenefitsUrl` still necessary? I see a few other instances in unit tests too